### PR TITLE
Add stateless validator monitoring support

### PR DESCRIPTION
**Pull request by a-thomas-22**
This fixes the servicemonitor and makes it more inline with the regular nitro config options

### CODE REVIEW

File: charts/nitro/templates/validator/service.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@ spec:
       protocol: TCP
       name: auth
   {{- end }}
-  {{- if $.Values.validator.statefulset.metrics.enabled }}
-    - port: {{ $.Values.validator.statefulset.metrics.port }}
-      targetPort: {{ $.Values.validator.statefulset.metrics.port }}
+  {{- if .Values.validator.configmap.data.metrics }}
```
Comment (gmas) [inline charts/nitro/templates/validator/service.yaml:21]: is `$` optional when referencing `.Values`

File: charts/nitro/templates/validator/servicemonitor.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if and $.Values.validator.enabled $.Values.validator.statefulset.metrics.enabled }}
+{{- if and $.Values.validator.enabled $.Values.validator.statefulset.serviceMonitor.enabled }}
```
Comment (gmas) [inline charts/nitro/templates/validator/servicemonitor.yaml:1]: Can't find the `.Values.validator.statefulset.serviceMonitor.enabled` key,  should this be `.Values.validator.serviceMonitor.enabled` ?

Comment (a-thomas-22) [reply charts/nitro/templates/validator/servicemonitor.yaml:1]: yes it should

**Review by gmas — APPROVED** [review]
LGTM :rocket:

**Pull request merged**